app/views/comments/_comment.html.erb

```diff
@@ -6,6 +6,13 @@
   <%= link_to user.name, user %>
   <%= time_ago_in_words(comment.created_at) %> ago.
   <br>
+    <%= pluralize(comment.upvote_count, "upvote") %>
+    <%= pluralize(comment.downvote_count, "downvote") %>
+    <br>
+    <%= render partial: 'vote/button',
+               locals: { votable: comment, value: "+1", dir: post_path(@post.id) } %>
+    <%= render partial: 'vote/button',
+               locals: { votable: comment, value: "-1", dir: post_path(@post.id) } %>
```

Comment:
better yet, should this be made into a partial, looks like it's used in a couple of places in your views